components/UserInput.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import { Space_Mono } from '@next/font/google'
+import { useState } from 'react';
+
+const smono = Space_Mono({
```

Comment:
Don't think you need this import if you use the `font-space-mono` Tailwind class.